app/controllers/application_controller.rb

```diff
@@ -13,6 +13,14 @@ def logged_in_user
     redirect_to login_path
   end
 
+  def logged_as_admin
+    return if current_user&.admin?
+
+    store_location
+    flash[:danger] = t "flash.login_plz"
```

Comment:
sao lại là login_plz 
Chỗ này em check chưa login hay check nó đã log in nhưng không phải admin em